modules/programs/mcfly.nix

```diff
@@ -74,20 +94,34 @@ in {
   };
 
   config = mkIf cfg.enable (mkMerge [
+    (mkIf cfg.mcfly-fzf.enable {
```

Comment:
I don't think the assertions are necessary, and indeed may cause false positives. For example, if you have `enableFishIntegration == false` but `programs.fish.enable = false`